TestApps/TestAppSwiftUI/AppDelegate.swift

```diff
@@ -29,13 +31,15 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
         let appState = application.applicationState
         MobileCore.setLogLevel(.trace)
         MobileCore.configureWith(appId: ENVIRONMENT_FILE_ID)
-        MobileCore.registerExtensions([
-            Identity.self,
-            Edge.self,
-            Assurance.self,
-            EdgeBridge.self,
-            Lifecycle.self
-        ], {
+
+        var extensions: [NSObject.Type] = [Edge.self, Identity.self, EdgeBridge.self, Lifecycle.self]
+
+        // MARK: TODO remove this once Assurance has tvOS support.
```

Comment:
Please remove this TODO. We will need to review and update the all our repos once we begin supporting assurance for tvOS.